03_intro_to_R_12_paths_in_code.Rmd

```diff
@@ -0,0 +1,139 @@
+
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Using Paths in Code
```

Comment:
Paths should come right before we talk about RMarkdown's use of relative paths. This may need some more editing.